modules/tinymce/src/themes/silver/main/ts/ui/context/ContextToolbar.ts

```diff
@@ -286,6 +288,13 @@ const register = (editor: Editor, registryContextToolbars: Record<string, Contex
       }
     });
 
+    editor.on('dragstart', () => {
+      isDragging = true;
+    });
+    editor.on('drop', () => {
```

Comment:
You need to act on dragend as well here since the state will not be reset if you drag content out of the editor.

Reply:
added with test